images/win/scripts/Tests/Android.Tests.ps1

```diff
@@ -66,4 +66,12 @@ Describe "Android SDK" {
     It "Additional tool <additionalToolVersion> is installed" -TestCases $additionalToolsTestCases {
         "$installedPackages" | Should -Match $additionalToolVersion
     }
+
+    It "LTS NDK is installed" {
+        "$androidInstalledPackages" | Should -Match $androidToolset.ndk.lts
+    }
+
+    It "Latest NDK is installed" {
+        "$androidInstalledPackages" | Should -Match $androidToolset.ndk.latest
+    }
 }
```

Comment:
nitpick: add a new line at the end of file

Reply:
Done.